### PR TITLE
Unused double in jsp wfly 4406

### PR DESCRIPTION
**Pull request by rhatlapa**
https://issues.jboss.org/browse/WFLY-4406
- Compiler for JSPs used by undertow (ECJ 4.4.1) crashes when compiling jsp which contains unused double value.
  Created test and proposed update of eclipse compiler from 4.4.1 to 4.4.2 (4.4.2 contains fix for it)

### CODE REVIEW

**Comment by ctomc** [comment]
:+1:

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsp/JspWithUnusedDoubleTestCase.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+@RunWith(Arquillian.class)
+@RunAsClient
+public class JspWithUnusedDoubleTestCase {
+    private static final String DEPLOYMENT = "jsp-with-unused-double.war";
+
+    @Deployment(name = DEPLOYMENT)
+    public static WebArchive deployment() {
+        WebArchive war = ShrinkWrap.create(WebArchive.class, DEPLOYMENT);
+        war.add(new UrlAsset(JspWithUnusedDoubleTestCase.class.getResource("jsp-with-unused-double.jsp")), "jsp-with-unused-double.jsp");
+        war.as(ZipExporter.class).exportTo(new File("/tmp", war.getName()), true);
```
Comment (ctomc) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsp/JspWithUnusedDoubleTestCase.java]: why? for debugging purposes? sure, but don't keep it in testsuite by default it just messes up with CI servers

Comment (rhatlapa) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsp/JspWithUnusedDoubleTestCase.java]: Thanks for noticing, it was for debugging purposes, I have removed it.

**Comment by rhatlapa** [comment]
Removed exporting the war (was only for debugging purposes), please retest.

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsp/JspWithUnusedDoubleTestCase.java (hunk 1 above)
Comment (ctomc) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsp/JspWithUnusedDoubleTestCase.java]: tnx

**Pull request merged**